target/linux/realtek/image/rtl930x.mk

```diff
@@ -95,12 +95,13 @@ define Device/xikestor_sks8310-8x
 endef
 TARGET_DEVICES += xikestor_sks8310-8x
 
-define Device/zyxel_xgs1210-12
+define Device/zyxel_xgs1210-12-v1
   SOC := rtl9302
   UIMAGE_MAGIC := 0x93001210
   ZYXEL_VERS := ABTY
   DEVICE_VENDOR := Zyxel
   DEVICE_MODEL := XGS1210-12
+  DEVICE_VARIANT := v1
```

Comment:
Please add this as a new line so `sysupgrade` allows updates from `xgs1210-12` to `xgs1210-12-v1`
`SUPPORTED_DEVICES += zyxel,xgs1210-12`

see for an example: https://github.com/openwrt/openwrt/blob/16ba3fc37920c2da10d086d228a86c76324c0620/target/linux/ramips/image/mt7621.mk#L419